blob/parser.go

```diff
@@ -77,54 +75,37 @@ func (p *parser) parse() (*Blob, error) {
 		return nil, fmt.Errorf("invalid shares amount. want:%d, have:%d", p.length, len(p.shares))
 	}
 
-	sequence, err := shares.ParseShares(p.shares, true)
+	blobs, err := gosquare.ParseBlobs(p.shares)
 	if err != nil {
 		return nil, err
 	}
 
-	// ensure that sequence length is not 0
-	if len(sequence) == 0 {
+	if len(blobs) == 0 {
```

Comment:
Yes, this is true. I'll modify the condition.

Reply:
Hmm, we still planned to return ErrBlobNotFound in case of singular `GetBlob`, while we removed it for multiple.

Reply:
@vgonkivs

Reply:
Discussed in DM. This does not remove ErrBlobNotFound usage entirely. Fun fact that this case shouldn't be possible in practice as it would be mean either validators made malformed block or GetSharesByNamespace proofs were wrong.